### ISSUE TITLE
plot.cvrisk x-axis tick labels

### ISSUE THREAD

**Issue by Almond-S** [boost-R/mboost#102]
The x-axis ticks are not labeled correctly as the attributes of the cvrisk-object x are dropped 
> here: https://github.com/boost-R/mboost/blob/47b008e3847f2c9c4d3f189f4aa5447d76508b14/R/crossvalidation.R#L107

as one can see for instance in this example:
```{r}
library(mboost)
data <- data.frame( y = rep(0:1, each = 6),
                    x = rep(0:1, 6))
model <- mboost(y ~ bols(x), data = data, control = boost_control(mstop = 5) )
cross <- cvrisk(model, grid = 0:5)
plot(cross)
```
To fix it, one would only have to store
`mstops <- attr(x, "mstop")` in advance and replace it bellow in

> https://github.com/boost-R/mboost/blob/47b008e3847f2c9c4d3f189f4aa5447d76508b14/R/crossvalidation.R#L118

**Comment by davidruegamer** [comment]
https://github.com/boost-R/mboost/issues/96

**Comment by Almond-S** [comment]
I think, it's a very related issue but different.

**Comment by thothorn** [comment]
Thanks, fixed in 2.9-3
